pkg/kubelet/kubelet_pods.go

```diff
@@ -1252,17 +1251,17 @@ func (pk *podKillerWithChannel) KillPod(podPair *kubecontainer.PodPair) {
 		_, runningPodExists = pk.podTerminationMap[uid]
 		if !runningPodExists {
 			fullname := podPair.RunningPod.Name
-			klog.V(4).Infof("marking running pod pending termination %q: %q", uid, fullname)
+			klog.V(4).InfoS("Marking running pod pending termination", "podName", fullname, "podUID", uid)
 			pk.podTerminationMap[uid] = fullname
 		}
 	}
 	if apiPodExists || runningPodExists {
 		if apiPodExists && runningPodExists {
-			klog.V(4).Infof("api pod %q and running pod %q is pending termination", podPair.APIPod.UID, podPair.RunningPod.ID)
+			klog.V(4).InfoS("Api pod and the running pod is pending termination", "apiPodUID", podPair.APIPod.UID, "runningPodUID", podPair.RunningPod.ID)
```

Comment:
drop the before running pod

initial string has error -> "are pending termination"